templates/advancement/scale-value-config.html

```diff
@@ -26,6 +26,7 @@
       </div>
       <p class="hint identifier-hint">
         {{{localize "DND5E.AdvancementScaleValueIdentifierHint" class=classIdentifier identifier=previewIdentifier}}}
+        <i name="identifier-hint-copy" class="far fa-clipboard"></i>
```

Comment:
This should be a link so it properly highlights for the users:

```suggestion
        <a class="identifier-hint-copy" title="{{localize 'DND5E.IdentifierCopy'}}">
          <i class="far fa-clipboard"></i>
        </a>
```

I also added a `title`:

```
"DND5E.IdentifierCopy": "Copy Identifier",
```